src/command_modules/azure-cli-acr/HISTORY.rst

```diff
@@ -5,6 +5,7 @@ Release History
 2.0.25
 ++++++
 Minor fix
+Add ACR Image Import command.
```

Comment:
Since you are adding the first relevant item for this release, you can remove "Minor fix" above. Also, recommend:
`* Added `acr image import` command.

Reply:
Thanks